Doc/library/smtplib.rst

```diff
@@ -496,12 +496,14 @@ An :class:`SMTP` instance has the following methods:
    specified in :rfc:`5322`\: *from_addr* is set to the :mailheader:`Sender`
    field if it is present, and otherwise to the :mailheader:`From` field.
    *to_addrs* combines the values (if any) of the :mailheader:`To`,
-   :mailheader:`Cc`, and :mailheader:`Bcc` fields from *msg*.  If exactly one
-   set of :mailheader:`Resent-*` headers appear in the message, the regular
-   headers are ignored and the :mailheader:`Resent-*` headers are used instead.
-   If the message contains more than one set of :mailheader:`Resent-*` headers,
-   a :exc:`ValueError` is raised, since there is no way to unambiguously detect
-   the most recent set of :mailheader:`Resent-` headers.
+   :mailheader:`Cc`, and :mailheader:`Bcc` fields from *msg*.  If there's no
+   *Date* header inside the message, ``send_message`` will add one to the data.
```

Comment:
```suggestion
   :mailheader:`Date` header inside the message, ``send_message`` will add one to the data.
```

Reply:
Thanks a lot for your time :) this is awesome.  I'm pushing the new version with the change as discussed.